src/extensions/surveys/surveys-utils.tsx

```diff
@@ -694,6 +694,20 @@ const getSurveyInteractionProperty = (survey: Survey, action: string): string =>
     return surveyProperty
 }
 
+export const hasWaitPeriodPassed = (
+    lastSeenSurveyDate: string | null,
+    waitPeriodInDays: number | undefined
+): boolean => {
+    if (!waitPeriodInDays || !lastSeenSurveyDate) {
+        return true
+    }
```

Comment:
i think either the method has to be renamed or the return has to be inverted.
`hasWaitPeriodPassed` returns true if no waitPeriodInDays, so its semantically inverted